rulebook.tex

```diff
@@ -1498,32 +1498,41 @@ \subsubsection{Production Plan}
 requiring 1 base, and two colors requiring no additional base at
 all. Products retrieved from the Storage Station may be used only for
 $C_0$ orders with a requested quantity equal to or greater than two.
-
-In each game, up to 10 orders will be placed.\footnote{There will up
-  to 9 orders in regular game time, and an additional order in the
-  case of overtime. The number can be adjusted during the tournament
-  by the Technical Committee, should this turn out to be necessary.}
+An order can be \emph{competitive} or \emph{non-competitive}. For a
+competitive order, only the team that delivers first will score points
+for the delivery, and it will score twice the amount of points. The
```

Comment:
```suggestion
for the delivery, and it will score higher (cf.~\reftab{tab:scoring-production}). The
```
This is wrong. Delivery in time 20 + 1st competitive 10 = 30, which is not twice the amount of points. This should just state "scores higher" and leave the actual value to the table, as to avoid inconsistencies if there are multiple sources of information for the same thing (we had plenty of those in the early years).

Reply:
Thanks for spotting this :+1: